libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/security/Encryptor.java

```diff
@@ -142,6 +154,18 @@ public static String decrypt(String data, String key) {
      * @return Decrypted data.
      */
     public static String decrypt(byte[] data, String key) {
+        return decrypt(data, key, new byte[16]);
```

Comment:
Why do we use `new byte[16]` here as a parameter of the constructor of `IvParameterSpec`?

Reply:
@Songroid It was [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/1611/files#diff-d64d9d238bf5bb327ddd3e159882e88bL338) before. Now it's passed in from the top. These methods exist for backward compatibility.